sematic/api/endpoints/metrics.py

```diff
@@ -0,0 +1,66 @@
+# Standard Library
+import enum
+import logging
+from typing import Dict, List, Optional, Type
+
+# Sematic
+from sematic.abstract_plugin import PluginScope
+from sematic.abstract_system_metric import AbstractSystemMetric
+from sematic.db.models.run import Run
+from sematic.db.models.user import User
+from sematic.metrics.metric_point import MetricPoint
+from sematic.metrics.run_count_metric import RunCountMetric
+from sematic.plugins.abstract_metrics_storage import get_metrics_storage_plugins
+from sematic.plugins.metrics_storage.sql.sql_metrics_storage import SQLMetricsStorage
+
+logger = logging.getLogger(__name__)
+
+
+class MetricEvent(enum.IntEnum):
+    run_created = 1
+
+
+_METRICS: Dict[MetricEvent, List[Type[AbstractSystemMetric]]] = {
+    MetricEvent.run_created: [
+        RunCountMetric,
+    ],
+}
+
+
+def save_event_metrics(
```

Comment:
Since there is currently no `internal` directory, and the `auth.py` and `events.py` modules already mix endpoints with utility functions, I will leave this here since this file will also contain endpoints in the next PR.